tests/manager/manager-executeQuery-001.phpt

```diff
@@ -73,7 +73,7 @@ object(MongoDB\Driver\Cursor)#%d (%d) {
       ["mode"]=>
       int(1)
       ["tags"]=>
-      array(0) {
+      object(stdClass)#%d (0) {
```

Comment:
> Why is this stdClass now? that looks weird?

I'm changing `php_phongo_read_preference_to_zval()`, `Server::getTags()`, `Server::getInfo()`, and `php_phongo_server_to_zval()` to use native array decoding, as we discussed earlier.